runtime-sdk/modules/evm/src/precompile/mod.rs

```diff
@@ -133,6 +133,7 @@ impl<Cfg: Config, B: EVMBackendExt> PrecompileSet for Precompiles<'_, Cfg, B> {
             (1, 1, 1) => sha512::call_sha512_256(handle),
             (1, 1, 2) => sha512::call_sha512(handle),
             (1, 1, 3) => subcall::call_subcall(handle, self.backend),
+            (1, 1, 4) => sha512::call_sha384(handle),
```

Comment:
I think you can also move this up (and reorder) as oasis specific group hasn't been deployed yet.

Reply:
It has been deployed to Testnet.

Reply:
I've stopped developing on networks because they broke their testnet. I know I have no right to be mad, but it's annoying nonetheless. I'd rather avoid that if the consequence is just a few numbers being out of order :)

Reply:
Agree, I forgot that it was already deployed.